clitest.sh

```diff
@@ -2,7 +2,7 @@
 # The intention is not to really validate the results of the command,
 # but to make note should any commands fail and other packaging failures.
 
-mvn clean  package -Dmaven.test.skip=true
+mvn clean package -Dmaven.test.skip=true
```

Comment:
Small stylistic changes like this -- especially in files that have not been otherwise modified -- should be avoided. Such changes cause git-blame to point at the latest stylistic change rather than the last change in functionality, which in turn makes browsing the change history much more difficult.